text/0020-block-limiters.md

```diff
@@ -0,0 +1,56 @@
+- Proposal Code Name: block-limiters
+- Start Date: 2019-10-09
+- NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
+
+# Summary
+[summary]: #summary
+
+We need to limit how many transactions are included in the block and how many transactions from the block are executed.
+Currently, our system is easily abusable, one can stall block production or make us lose receipts.
+
+# Motivation
+[motivation]: #motivation
+
+We have several problems at the moment:
+
+* Currently our block production can be easily abused. For instance one can submit 20 transactions each of which
+deploys a 1MiB contract. Together these transactions would take > 1 sec to process, however block producer will happily
+include them into the block and when the next block is produced (potentially by a different block producer) node will choke on
+processing it;
+
+* Currently we have a limit on how many transactions+receipts we include in the block, but if we have number of receipts
```

Comment:
Actually this is not true. We only have a limit on the number of transactions and include all the receipts.